src/sidebar/components/test/PaginationNavigation-test.js

```diff
@@ -0,0 +1,159 @@
+import { mount } from 'enzyme';
+import { act } from 'preact/test-utils';
+
+import PaginationNavigation, { $imports } from '../PaginationNavigation';
+
+describe('PaginationNavigation', () => {
+  let fakeOnChangePage;
+  let fakePageNumberOptions;
+
+  let container;
+
+  const findButton = (wrapper, title) =>
+    wrapper.find('button').filterWhere(n => n.props().title === title);
+
+  const createComponent = (props = {}) => {
+    return mount(
+      <PaginationNavigation
+        currentPage={1}
+        onChangePage={fakeOnChangePage}
+        totalPages={10}
+        {...props}
+      />,
+      { attachTo: container }
+    );
+  };
+
+  beforeEach(() => {
+    // This extra element is necessary to test automatic `focus`-ing
+    // of the component's `input` element
+    container = document.createElement('div');
+    document.body.appendChild(container);
+
+    fakeOnChangePage = sinon.stub();
+    fakePageNumberOptions = sinon.stub().returns([1, 2, 3, 4, null, 10]);
+
+    $imports.$mock({
+      '../util/pagination': { pageNumberOptions: fakePageNumberOptions },
+    });
+  });
+
+  afterEach(() => {
+    $imports.$restore();
+  });
```

Comment:
`container` should be removed from the DOM here. Otherwise instances of this component will accumulate in the DOM for every test.

Reply:
`container` has been removed entirely.